views/eApostilles/openEApp.ejs

```diff
@@ -5,6 +5,24 @@
         <%- partial ('../partials/inner-header.ejs')%>
     </div>
 
+    <% if (applicationStatus === 'Completed' && allDocumentsRejected || applicationStatus === 'Completed' && someDocumentsRejected) { %>
```

Comment:
Could be shortened to?

```html
<% if (applicationStatus === 'Completed' && (allDocumentsRejected || someDocumentsRejected)) { %>
```

Reply:
Ah nice that does make sense, I'll give it a go